src/components/time-entry-add/TimeEntryAdd.jsx

```diff
@@ -47,108 +47,111 @@ const TimeEntryAdd = ({ addFormData }) => {
-            htmlFor="client"
-          >
-            <p className={styles.labelText}>CLIENT</p>
-            <input
-              className={`${styles.input} ${
-                validity.client === false ? styles.invalidInput : ''
-              }`}
-              id="client"
-              maxLength="30"
-              minLength="2"
-              name="client"
-              onBlur={handleBlur}
-              onChange={({ target }) => setClient(target.value)}
-              value={client}
-              required
-            />
-          </label>
-
-          <label
-            className={`${styles.label} ${styles.labelLarge}`}
-            htmlFor="activity"
-          >
-            <p className={styles.labelText}>ACTIVITY</p>
-            <input
-              className={`${styles.input} ${
-                validity.activity === false ? styles.invalidInput : ''
-              }`}
-              id="activity"
-              maxLength="30"
-              minLength="2"
-              name="activity"
-              onBlur={handleBlur}
-              onChange={({ target }) => setActivity(target.value)}
-              required
-              value={activity}
-            />
-          </label>
-          <label
-            className={`${styles.label} ${styles.labelMedium} ${styles.date}`}
-            htmlFor="date"
-          >
-            <p className={styles.labelText}>DATE</p>
-            <input
-              className={styles.input}
-              id="date"
-              name="date"
-              onChange={({ target }) => setDate(target.value)}
-              required
-              type="date"
-              value={date}
-            />
-          </label>
-
-          <label
-            className={`${styles.label} ${styles.labelSmall} ${styles.labelHalfWidth}`}
-            htmlFor="startTime"
-          >
-            <p className={styles.labelText}>FROM</p>
-            <input
-              className={`${styles.input} ${styles.timeStamp}`}
-              id="startTime"
-              name="startTime"
-              onChange={({ target }) => setStartTime(target.value)}
-              type="time"
-              value={startTime}
-            />
-          </label>
-          <label
-            className={`${styles.label} ${styles.labelSmall} ${styles.labelHalfWidth}`}
-            htmlFor="endTime"
-          >
-            <p className={styles.labelText}>TO</p>
-            <input
-              className={`${styles.input} ${styles.timeStamp}`}
-              id="endTime"
-              name="endTime"
-              onChange={({ target }) => setEndTime(target.value)}
-              type="time"
-              value={endTime}
-            />
-          </label>
-          <button
-            className={styles.addButton}
-            disabled={validity.form !== true}
-            type="submit"
-          >
-            Add
-          </button>
-        </form>
-      )}
+      <form
```

Comment:
Why did you remove the toggling behavior?

Reply:
I removed this behavior only for desktop, since it should always be visible there.

Reply:
Like this it seems to be gone on mobile as well? You've removed the condition to show and hide the form.

Reply:
Two different buttons toggle the form visibility state. That state is used to trigger a CSS class in TimeEntryAdd called `formVisible`.